app/services/teachers/current_schools.rb

```diff
@@ -0,0 +1,14 @@
+module Teachers
+  class CurrentSchools
+    def initialize(trn:)
+      @trn = trn
+    end
+
+    def ect_at
+      teacher = Teacher.find_by(trn: @trn)
```

Comment:
If we have more than one method in here it could make sense to set the `@teacher` in the initializer instead of the TRN

```ruby
@teacher = Teacher.find_by(trn:)
```

Reply:
Thank you, i've also updated as part of this commit: https://github.com/DFE-Digital/register-early-career-teachers-public/pull/554/commits/c5adfa43159babdd439ad94fdbc5cc3cfccc467c